### ISSUE TITLE
Feature: Support for multiple email addresses synced from LDAP

### ISSUE THREAD

**Issue by stertingen** [go-gitea/gitea#12301]
- Gitea version (or commit ref): 1.12.2 built with GNU Make 4.2.1, go1.14.4 : bindata, sqlite, sqlite_unlock_notify
- Git version: 2.24.3
- Operating system: official docker container
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description

We have a central LDAP server for our users who may have multiple email addresses, which are also stored on the LDAP server. I'd like to propose that the LDAP authentication source shall look for multiple email addresses of that user and add these to the user profile.

Each user should be able to add and remove more addresses on their own, but addresses synced from the LDAP server should only be removable by removing them from the LDAP server.

**Comment by urz-hgw** [comment]
I maybe have some additional use case for this feature request:

We also have multiple email addresses per user in our LDAP and a user can select a primary email address which is stored in a separate LDAP attribute which I currently use in the LDAP configuration in Gitea.

When a user adds another email address in Gitea  (either the other one from LDAP or an external email address) and set this new one as the primary email address in Gitea, the log throws a warning at LDAP sync

```
gitea-1    | 2026/02/09 20:47:04 .../source/ldap/source_sync.go:177:(*Source).Sync() [E] SyncExternalUsers[LDAP]: Error updating user X primary email primaryemail@ldap: e-mail already in use [email: primaryemail@ldap]
```

So basically the sync throws an error when the email address in LDAP is NOT the primary email address for a user in Gitea.

This issue should also be solved by adding this feature.

Best regards
Daniel
